.travis.yml

```diff
@@ -1,10 +1,21 @@
-dist: xenial
+matrix:
+  include:
+    - name: "reStructuredText syntax check"
+      dist: xenial
+      language: python
+      python:
+        - 3.7
+      install:
+        - pip3 install rstcheck==3.3.1
+      script:
+        - rstcheck -r source/ --ignore-language c,python,java --report error
 
-language: python
-
-python:
-  - 3.7
-
-install: pip3 install rstcheck==3.3.1
-
-script: rstcheck -r source/ --ignore-language c,python,java --report error
+    - name: "JSON regeneration check"
+      dist: trusty
+      language: node_js
+      node_js:
+        - 12
+      install:
+        - npm install -g js-yaml
+      script:
+        - cd source && make && git diff --exit-code
```

Comment:
I assume this kills two birds in one stone, by both requiring the build to complete _and_ expecting no new JSON changes?

Reply:
If by 'requiring the build to complete' you mean requiring that the `make` command executes successfully, then yes.